README.md

```diff
@@ -1 +1,114 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers (JS, PHP, Python, Ruby, Go, Java, .NET)
+- [ ] mysqldump
+- [ ] grafana
+- [ ] BI (Tableau, PowerBI, pick one)
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions with basic MySQL auth implementation
+- [ ] Audit logs (connections, queries)
+
+### Get web interfaces enterprise ready [APPS]
+
+- [ ] gitbase-web SSO (protocol TBD)
+- [ ] bblfsh-web SSO (protocol TBD)
+
+### Gitbase research [DR]
+
+- [ ] Support updates in backing data (indexes, etc)
+- [ ] Usage of caches for UAST and others
+
+### Launch gitbase-spark-connector [DR]
+
+- [ ] Launch gitbase-spark-connector community beta version.
+- [ ] Launch gitbase-spark-connector enterprise beta version.
+
+### Release necessary language support for enterprise [LA]
+
+- [ ] C# driver
+- [ ] C/C++ driver
+
+### Get bblfsh ready for wide language support of our use cases [LA]
+
+- Finish migration to v2 for all supported client languages (JVM, Python, Go)
+- Deprecate v1
+- Fix positions in all alpha+ drivers
+
+### Have an available demo cluster [INFRA]
+
+- [ ] Deploy a demo cluster for 'Code as Data' that we can use for demos and 3rd parties
+
+### Define a delivery strategy for the enterprise
+
+- [ ] Test deployment for faux-customer [INFRA][ENG]
+
+### Get Gemini production ready [APPS]
+
+- [ ] Release a new version with the integration of function similarity
+
+## Improve the 'Code as Data' individual user experience
+
+- [ ] Move engine maintainership to apps [APPS]
+
+## Improve quality across all products [INFRA][DR]
+
+- [ ] Regression test platform [INFRA]
+- [ ] Automated regression tests for borges [DR]
+- [ ] Automated regression tests for gitbase [DR]
+- [ ] Conduct two empathy sessions for cross-team documentation and user experience improvements [ENG][QA]
+
+## Release a new version of PGA [INFRA][APPS]
+
+- [ ] Incremental PGA updates [APPS]
+- [ ] Automate PGA generation [APPS][INFRA]
+- [ ] Release PGA v2 [APPS]
+
+## Bring 'Assisted Code Review' to developers in the open-source community
+
+### Get lookout production ready [APPS]
+
+- [ ] Extend lookout end user documentation
```

Comment:
would it worth to be specific here, same as in borges below, and mention that docs should be using Gitbook?